ckan/templates/organization/snippets/organization_form.html

```diff
@@ -57,7 +57,7 @@
   <div class="form-actions">
     {% block delete_button %}
       {% if h.check_access('organization_delete', {'id': data.id})  %}
-        <a class="btn btn-danger pull-left" href="{% url_for controller='organization', action='delete', id=data.id %}" data-module="confirm-action" data-module-content="{{ _('Are you sure you want to delete this Organization? This will delete all the public and private datasets belonging to this organization.') }}">{% block delete_button_text %}{{ _('Delete') }}{% endblock %}</a>
+        <a class="btn btn-danger pull-left" href="{% url_for controller='organization', action='delete', id=data.id %}" data-module="confirm-action" data-module-content="{{ _('Are you sure you want to delete this Organization? Note*: Deleting cannot be performed while public and private datasets are belonging to this organization.') }}">{% block delete_button_text %}{{ _('Delete') }}{% endblock %}</a>
```

Comment:
See my comment above regarding `ckan.auth.create_unowned_dataset=True`. If that is the case the message should read "Note: All datasets from this organization won't belong to any organization after it is deleted" or something similar.